cryptocurrency/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: Cryptocurrency
+permalink: /cryptocurrency/
+---
+
+Cryptocurrencies, most popularly Bitcoin, are becoming a popular alternative to credit cards payments online. Cryptocurrencies are a digital currency based on cryptography and provide some benefits such as anonymous transactions and lower transaction fees. 
```

Comment:
its pure what I've seen / heard, from Dell, Ostock.com accepting it and all my bitcoin friends. # of btc transactions has grown significantly over time, but it may just be a "Silicon Valley bubble" thing https://blockchain.info/charts/n-transactions?timespan=all&showDataPoints=false&daysAverageString=1&show_header=true&scale=0&address=